tests/pubsub/testgossipsubcontrolmessages.nim

```diff
@@ -444,7 +444,7 @@ suite "GossipSub Control Messages":
 
     # Then B doesn't relay the message to C.
     checkUntilTimeout:
-      nodes[1].mesh.getOrDefault(topic).anyIt(it.iDontWants[^1].len == 1)
+      nodes[1].mesh.getOrDefault(topic).anyIt(it.iDontWants.anyIt(it.len == 1))
```

Comment:
Curious to understand why the change from `[^1]` to `anyIt`, is this because there's no guarantee for the messages to arrive in the same heartbeat?